config.json

```diff
@@ -3,53 +3,423 @@
+      ]
+    },
+    {
+      "slug": "gigasecond",
+      "difficulty": 1,
+      "topics": [
+        "crates",
+        "math"
+      ]
+    },
+    {
+      "slug": "leap",
+      "difficulty": 1,
+      "topics": [
+        "math",
+        "booleans",
+        "conditionals"
+      ]
+    },
+    {
+      "slug": "raindrops",
+      "difficulty": 1,
+      "topics": [
+        "case (or `format`)",
+        "Mutable string"
+      ]
+    },
+    {
+      "slug": "bob",
+      "difficulty": 1,
+      "topics": [
+        "chars",
+        "string functions"
+      ]
+    },
+    {
+      "slug": "beer-song",
+      "difficulty": 1,
+      "topics": [
+        "case",
+        "string concatenation",
+        "vector (optional)",
+        "loop"
+      ]
+    },
+    {
+      "slug": "difference-of-squares",
+      "difficulty": 1,
+      "topics": [
+        "fold & map"
+      ]
+    },
+    {
+      "slug": "grains",
+      "difficulty": 1,
+      "topics": [
+        "math",
+        "panic"
+      ]
+    },
+    {
+      "slug": "hamming",
+      "difficulty": 1,
+      "topics": [
+        "Result"
+      ]
+    },
+    {
+      "slug": "pascals-triangle",
+      "difficulty": 1,
+      "topics": [
+        "Math",
+        "Vec",
+        "Index (optional)"
+      ]
+    },
+    {
+      "slug": "scrabble-score",
+      "difficulty": 1,
+      "topics": [
+        "chaining higher-order functions",
+        "HashMap (optional)"
+      ]
+    },
+    {
+      "slug": "pangram",
+      "difficulty": 1,
+      "topics": [
+        "filter",
+        "ascii (optional)"
+      ]
+    },
+    {
+      "slug": "nucleotide-count",
+      "difficulty": 1,
+      "topics": [
+        "filter",
+        "entry api",
+        "mutablity",
+        "match"
+      ]
+    },
+    {
+      "slug": "word-count",
+      "difficulty": 1,
+      "topics": [
+        "hashmap",
+        "str vs string",
+        "chars",
+        "entry api"
+      ]
+    },
+    {
+      "slug": "atbash-cipher",
+      "difficulty": 1,
+      "topics": [
+        "str vs string",
+        "primitive types",
+        "iterators",
+        "chars",
+        "ascii"
+      ]
+    },
+    {
+      "slug": "etl",
+      "difficulty": 1,
+      "topics": [
+        "btree"
+      ]
+    },
+    {
+      "slug": "acronym",
+      "difficulty": 1,
+      "topics": [
+        "map",
+        "flat_map",
+        "filter",
+        "loops",
+        "Vec"
+      ]
+    },
+    {
+      "slug": "sieve",
+      "difficulty": 1,
+      "topics": [
+        "vector",
+        "map",
+        "while let (optional)"
+      ]
+    },
+    {
+      "slug": "rna-transcription",
+      "difficulty": 1,
+      "topics": [
+        "match",
+        "struct",
+        "str vs string"
+      ]
+    },
+    {
+      "slug": "triangle",
+      "difficulty": 1,
+      "topics": [
+        "Math",
+        "Struct",
+        "Enum",
+        "Trait",
+        "Box -- depending on implementation"
```

Comment:
I'd remove Trait, Enum and Box from there. I think I added those when I was trying a very different solution. Students might still use that approach, but it will be far from standard.